rakelib/artifacts.rake

```diff
@@ -580,6 +584,15 @@ namespace "artifact" do
     File.join(basedir, "pkg", "pipelines.yml").tap do |path|
       dir.input("#{path}=/etc/logstash")
     end
+    File.join(basedir, "pkg", "service_templates", "systemd", "lib", "systemd", "system", "logstash.service").tap do |path|
+      dir.input("#{path}=/lib/systemd/system")
+    end
+    File.join(basedir, "pkg", "service_templates", "sysv", "etc", "init.d", "logstash").tap do |path|
+      dir.input("#{path}=/etc/init.d")
+    end
+    File.join(basedir, "pkg", "service_templates", "sysv", "etc", "default", "logstash").tap do |path|
+      dir.input("#{path}=/etc/default")
+    end
```

Comment:
as I understand it always install `sysv` and `systemd` init files independently of which service manager is used by the target system, so does it install files that potentially are not used by the target distribution?

Reply:
Actually there's no harm installing both because sysv has been pretty much replaced in every single OS, but all the underlying files are being kept. That said, the system is smart enough that it will always use systemd units even if there's sysv init files.

Please do keep in mind that all products already ship sysv init scripts alongside systemd units (kibana, filebeat, etc).